clover.xml

```diff
@@ -0,0 +1,468 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think, you can add this file to `.gitignore`

Reply:
Done. Sorry I forgot it!